### DISCUSSION TITLE
Open keymap view. How?

### DISCUSSION THREAD

**Discussion by unphased** [sxyazi/yazi#1520] [1. Q&A]
I'll need to set this up so i can open yazi and toggle the keymap view: https://github.com/sxyazi/yazi/issues/79 to learn how to use it most effectively. 

I have reviewed the built in keymaps https://github.com/sxyazi/yazi/blob/shipped/yazi-config/preset/keymap.toml and found nothing relevant in there. So please can somebody share how to enable the feature so I can begin to move all of my `lf` workflows over to `yazi` as it is a superior piece of software already. 

Thanks

**Comment by unphased** [comment]
I can recommend to the project that language in the quick start page be added to indicate that help is synonymous with a keymap listing, because it took me nearly an hour to learn how to pull this up. In my mind I wanted to skip reading any help document and look at the list of keymaps. Lo and behold the former doesn't even exist. Badass. It can be done by hitting tilde (shift+backtick) or F1.

**Comment by xfzv** [comment]
> I have reviewed the built in keymaps https://github.com/sxyazi/yazi/blob/shipped/yazi-config/preset/keymap.toml and found nothing relevant in there.

You clearly missed it:

https://github.com/sxyazi/yazi/blob/4112bf451faa995f34763b2de3efb5aa715d107a/yazi-config/preset/keymap.toml#L149-L151
https://github.com/sxyazi/yazi/blob/4112bf451faa995f34763b2de3efb5aa715d107a/yazi-config/preset/keymap.toml#L171-L173

**Reply by unphased** [reply]
I opened help once I saw that in the keymap.toml, yeah, but i "missed" it the first time because I wrongly assumed help != keymaps when that is not the case. All I'm saying is for those who will make the same mistake as me, the documentation should clearly state that F1 or tilde actually pull up keymaps, to motivate people like me (who are looking for keymaps and not help) to open it. hope that makes sense.
